.github/workflows/build-images.yml

```diff
@@ -29,8 +29,18 @@ jobs:
         uses: docker/setup-buildx-action@v2
         with:
           install: true
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        if: (github.ref != 'refs/heads/master' && github.event_name != 'pull_request') || github.event_name == 'schedule'
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        if: (github.ref != 'refs/heads/master' && github.event_name != 'pull_request') || github.event_name == 'schedule'
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ github.token }}
       - name: Docker Build
-        env:
-          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
-          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
I'm very happy to see that we don't have to expose these to the whole environment anymore! :+1:

Reply:
I think it would be better to use privatebin directly in this case, since it always need to be it and github.actor will never be "privatebin", see here: https://docs.github.com/en/actions/learn-github-actions/contexts